from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_ProfileWindowWidget(object):
    def setupUi(self, ProfileWindowWidget):
        ProfileWindowWidget.setObjectName("ProfileWindowWidget")
        ProfileWindowWidget.resize(400, 356)
        self.nameLabel = QtWidgets.QLabel(ProfileWindowWidget)
        self.nameLabel.setGeometry(QtCore.QRect(20, 10, 241, 31))
        font = QtGui.QFont()
        font.setPointSize(14)
        self.nameLabel.setFont(font)
        self.nameLabel.setObjectName("nameLabel")
        self.contractsTableWidget = QtWidgets.QTableWidget(ProfileWindowWidget)
        self.contractsTableWidget.setGeometry(QtCore.QRect(20, 140, 361, 151))
        self.contractsTableWidget.setObjectName("contractsTableWidget")
        self.contractsTableWidget.setColumnCount(0)
        self.contractsTableWidget.setRowCount(0)
        self.contractsTableWidget.verticalHeader().setVisible(False)
        self.passwordLabel = QtWidgets.QLabel(ProfileWindowWidget)
        self.passwordLabel.setGeometry(QtCore.QRect(20, 50, 241, 31))
        font = QtGui.QFont()
        font.setPointSize(14)
        self.passwordLabel.setFont(font)
        self.passwordLabel.setObjectName("passwordLabel")
        self.namePushButton = QtWidgets.QPushButton(ProfileWindowWidget)
        self.namePushButton.setGeometry(QtCore.QRect(290, 10, 93, 31))
        self.namePushButton.setObjectName("namePushButton")
        self.passwordPushButton = QtWidgets.QPushButton(ProfileWindowWidget)
        self.passwordPushButton.setGeometry(QtCore.QRect(290, 50, 93, 31))
        self.passwordPushButton.setObjectName("passwordPushButton")
        self.exitPushButton = QtWidgets.QPushButton(ProfileWindowWidget)
        self.exitPushButton.setGeometry(QtCore.QRect(290, 310, 93, 31))
        self.exitPushButton.setObjectName("exitPushButton")
        self.phoneLabel = QtWidgets.QLabel(ProfileWindowWidget)
        self.phoneLabel.setGeometry(QtCore.QRect(20, 90, 241, 31))
        font = QtGui.QFont()
        font.setPointSize(14)
        self.phoneLabel.setFont(font)
        self.phoneLabel.setObjectName("phoneLabel")
        self.phonePushButton = QtWidgets.QPushButton(ProfileWindowWidget)
        self.phonePushButton.setGeometry(QtCore.QRect(290, 90, 93, 31))
        self.phonePushButton.setObjectName("phonePushButton")
        self.saleLabel = QtWidgets.QLabel(ProfileWindowWidget)
        self.saleLabel.setGeometry(QtCore.QRect(20, 310, 241, 31))
        font = QtGui.QFont()
        font.setPointSize(14)
        self.saleLabel.setFont(font)
        self.saleLabel.setObjectName("saleLabel")

        self.retranslateUi(ProfileWindowWidget)
        QtCore.QMetaObject.connectSlotsByName(ProfileWindowWidget)

    def retranslateUi(self, ProfileWindowWidget):
        _translate = QtCore.QCoreApplication.translate
        ProfileWindowWidget.setWindowTitle(_translate("ProfileWindowWidget", "Профиль"))
        self.nameLabel.setText(_translate("ProfileWindowWidget", "Name"))
        self.passwordLabel.setText(_translate("ProfileWindowWidget", "Password"))
        self.namePushButton.setText(_translate("ProfileWindowWidget", "Изменить"))
        self.passwordPushButton.setText(_translate("ProfileWindowWidget", "Изменить"))
        self.exitPushButton.setText(_translate("ProfileWindowWidget", "Выйти"))
        self.phoneLabel.setText(_translate("ProfileWindowWidget", "Phone"))
        self.phonePushButton.setText(_translate("ProfileWindowWidget", "Изменить"))
        self.saleLabel.setText(_translate("ProfileWindowWidget", "Sale"))
